Compiler/BackEnd/SymbolicJacobian.mo

```diff
@@ -1579,6 +1579,85 @@ algorithm
   end if;
 end createBipartiteGraph;
 
+public function createFMIModelDerivatives
+"This function genererate the stucture output and the
+ partial derivatives for FMI, which are basically the jacobian matrices.
+ author: wbraun"
+  input BackendDAE.BackendDAE inBackendDAE;
+  output BackendDAE.SymbolicJacobians outJacobianMatrixes = {};
+  output DAE.FunctionTree outFunctionTree;
+protected
+  BackendDAE.BackendDAE backendDAE,emptyBDAE;
+  BackendDAE.EqSystem eqSyst;
+  Option<BackendDAE.SymbolicJacobian> outJacobian;
+
+  list<BackendDAE.Var> varlst, knvarlst, states, inputvars, outputvars, paramvars, indepVars, depVars;
+
+  BackendDAE.Variables v,globalKnownVars,statesarr,inputvarsarr,paramvarsarr,outputvarsarr, depVarsArr;
+
+  BackendDAE.SparsePattern sparsePattern;
+  BackendDAE.SparseColoring sparseColoring;
+
+  DAE.FunctionTree funcs, functionTree;
+
+  BackendDAE.ExtraInfo ei;
+  FCore.Cache cache;
+  FCore.Graph graph;
+algorithm
+try
+	// for now perform on collapsed system
```

Comment:
Not since we added the pre-commit hook that replaces all tabs. You shouldn't disable it ;)